File: C#/Problems/Problems/Problems.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Problems
{
    class Problems
    {

        // Problem 1
        public bool CanListSumToK(int[] list, int k)
        {
            HashSet<int> hashSet = new HashSet<int>();
            foreach (int i in list)
            {
                if (hashSet.Contains(i))
                {
                    return true;
                }
                else
                {
                    hashSet.Add(k-i);
                }
            }


            return false;
        }

    }
}

File: Golang/Problems.go
package main

//Problem 1
func CanListSumToK(list []int, k int) bool{
	hashset := map[int]bool{}

	for _, v := range list{
		if hashset[v]{
			return true
		} else {
			hashset[k-v] = true;
		}
	}
	return false;
}
